google-cloud-text_to_speech/Gemfile

```diff
@@ -2,6 +2,12 @@ source "https://rubygems.org"
 
 gemspec
 
+gem "google-cloud-core", path: "../google-cloud-core"
+gem "google-cloud-env", path: "../google-cloud-env"
+gem "google-cloud-errors", path: "../google-cloud-errors"
+gem "google-cloud-text_to_speech-v1", path: "../google-cloud-text_to_speech-v1"
+gem "google-cloud-text_to_speech-v1beta1", path: "../google-cloud-text_to_speech-v1beta1"
+
 gem "rake"
```

Comment:
rake and the minitest pin are still here, do you want to remove them now?

Reply:
@quartzmo Oh right, I forgot about those. I'll get them updated. Thanks for catching it.